Zoo/Tour/Visitor.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Zoo.Util.Enum;
+
+namespace Zoo.Tour
+{
+    internal sealed class Visitor
+    {
+        public string? Name { get; init; }
+
+        private readonly Dictionary<ZonesType, string> places = new()
+        {
+            { ZonesType.BirdZone, "Birds Zone" },
+            { ZonesType.OrdinaryLandZone, "Land Zone" },
+            { ZonesType.SeaCreatureZone, "Sea Zone" },
+            { ZonesType.AmphibiaZone, "Amphibia Zone" }
+        };
+
+
+        public Visitor(string Name)
+        {
+            Random random = new();
+            ZonesType randomZone = ZonesType.None;
+
+            int value = random.Next(1, 4);
```

Comment:
value is? I can look over the code and understand what it represents. But why should I? Code is supposed to be clear, or in pro terms- CLEAN CODE.